src/syntax_highlight/mod.rs

```diff
@@ -0,0 +1,9 @@
+#[cfg(all(feature="syntax-highlight", not(windows)))]
+mod syntect;
+#[cfg(all(feature="syntax-highlight", not(windows)))]
+pub use self::syntect::*;
+
+#[cfg(any(not(feature="syntax-highlight"), windows))]
+mod null;
+#[cfg(any(not(feature="syntax-highlight"), windows))]
+pub use self::null::*;
```

Comment:
Heh.  I started doing this with a patch I'm tinkering with for the dependency injection idea we discussed over on #200.  Glad to see I was on the same track with this idea.